Documentation/Makefile

```
@@ -45,7 +45,32 @@ copy-api:
 	@$(ECHO_GEN)_api
 	$(QUIET)cp -r ../api _api
 
-epub latex html: builder-image copy-api
+HELM_VALUES = helm-values.rst
+
+HELM_DOCS_VERSION := v1.3.0
+HELM_DOCS_SHA := 12f065115654c9a1d6819b938487d327f1fdb5c32bdc2a5c74afef22ed68f5ca
+HELM_DOCS_IMAGE := docker.io/jnorwood/helm-docs:$(HELM_DOCS_VERSION)@sha256:$(HELM_DOCS_SHA)
+HELM_DOCS_ROOT_PATH := /src
+HELM_DOCS_CHARTS_DIR := $(HELM_DOCS_ROOT_PATH)/install/kubernetes
+HELM_DOCS_OUTPUT_DIR := $(HELM_DOCS_ROOT_PATH)/Documentation
+HELM_DOCS := $(DOCKER_CTR) -v $(PWD)/..:$(HELM_DOCS_ROOT_PATH) $(HELM_DOCS_IMAGE)
```

```suggestion
HELM_DOCS := $(DOCKER_CTR) -v $(CURDIR)/..:$(HELM_DOCS_ROOT_PATH) $(HELM_DOCS_IMAGE)
```
Or `make -C Documentation` from the root of the repo does not work.